common/src/main/python/dlpx/virtualization/common/util.py

```diff
@@ -0,0 +1,136 @@
+#
+# Copyright (c) 2021 by Delphix. All rights reserved.
+#
+
+
+"""
+Utility functions to convert between unicode and bytes.
+"""
+
+import six
+
+
+def to_bytes(string, encoding=None):
```

Comment:
The default value of `encoding` here is `None`, but then later on, we programmatically will set `encoding` to `utf-8` if it's `None`.

Would it be clearer to just set the default to `utf-8` here, so that people can see from the signature what the default value is?
```
def to_bytes(string, encoding="utf-8"):
```